src/grammar.pegjs

```diff
@@ -75,20 +75,20 @@ AbsolutePath
 }
 
   Root
-    = PosixRoot
-    / WindowsRoot
+    = WindowsRoot
+    / PosixRoot
 
-    PosixRoot
-      = Sep {
+    WindowsRoot
+      = Sep ? drive:[A-Za-z] ":" Sep {
         return {
-          drive: null
+          drive: drive.toLowerCase() + ':'
         }
       }
 
-    WindowsRoot
-      = drive:[A-Za-z] ":" Sep {
+    PosixRoot
+      = Sep {
         return {
-          drive: drive.toLowerCase() + ':'
+          drive: null
```

Comment:
This diff looks stupid because I re-ordered it. Here's the actual diff:

```diff
     WindowsRoot
-      = drive:[A-Za-z] ":" Sep {
+      = Sep ? drive:[A-Za-z] ":" Sep { {
         return {
```